Resources/Prototypes/Language/Species-Specific/harpy.yml

```
@@ -0,0 +1,116 @@
+- type: language
+  id: ValyrianStandard
+  speech:
+    fontId: Cambria
+    color: "#008ecc"
+  obfuscation:
+    !type:SyllableObfuscation
+    minSyllables: 2
+    maxSyllables: 5
+    replacement:
+    - ш
```

yamllint

[error] wrong indentation: expected 6 but found 4

(indentation)